java/core/src/test/java/com/microsoft/bond/testmodel/MyEnum.java

```diff
@@ -78,6 +78,30 @@ public static MyEnum get(int value) {
         }
     }
 
+    public static MyEnum parse(String str) {
+        if (str == null) {
+            throw new IllegalArgumentException("Argument 'str' must not be null.");
+        } else if (str.equals("White")) {
+            return White;
+        } else if (str.equals("Yellow")) {
+            return Yellow;
+        } else if (str.equals("Orange")) {
+            return Orange;
+        } else if (str.equals("Purple")) {
+            return Purple;
+        } else if (str.equals("Blue")) {
+            return Blue;
+        } else if (str.equals("Green")) {
+            return Green;
+        } else if (str.equals("Brown")) {
+            return Brown;
+        } else if (str.equals("Black")) {
+            return Black;
+        } else {
+            throw new IllegalArgumentException("Invalid 'MyEnum' enum value: '" + str + "'.");
```

Comment:
Consider "Unknown" instead of "Invalid". It may be that this schema is old and doesn't know about a new value.